skyportal/models.py

```diff
@@ -1510,6 +1510,8 @@ def get_if_owned_by(cls, ident, user, options=[]):
 
         return annotation
 
+    __table_args__ = (UniqueConstraint('obj_id', 'origin'),)
```

Comment:
Why not just specify `unique=True` in `sa.Column` above?

Reply:
I believe this is here to define that within a given candidate's annotations, origin must be unique. Different candidates can have the same origin so we don't want to make the origin column itself unique.